services/cd-service/pkg/repository/transformer.go

```diff
@@ -124,6 +124,23 @@ func UpdateDatadogMetrics(state *State) error {
 	return nil
 }
 
+func SendRegularlyDatadogMetrics(repo Repository, interval int, callBack func(repository Repository)) {
```

Comment:
Either
```suggestion
func RegularlySendDatadogMetrics(repo Repository, interval int, callBack func(repository Repository)) {
```
OR:
```suggestion
func SendDatadogMetricsRegularly(repo Repository, interval int, callBack func(repository Repository)) {
```

Reply:
Fixed, function is now RegularlySendDatadogMetrics